hphp/runtime/vm/jit/relocation.h

```diff
@@ -66,22 +66,16 @@ struct RelocationInfo {
   std::set<TCA> addressImmediates;
 };
 
-namespace x64 {
-
+/*
+ * Wrappers
+ */
 void adjustForRelocation(RelocationInfo&);
 void adjustForRelocation(RelocationInfo& rel, TCA srcStart, TCA srcEnd);
 void adjustCodeForRelocation(RelocationInfo& rel, CGMeta& fixups);
-void adjustMetaDataForRelocation(RelocationInfo& rel,
-                                 AsmInfo* asmInfo,
-                                 CGMeta& fixups);
+void adjustMetaDataForRelocation(RelocationInfo&, AsmInfo*, CGMeta&);
 void findFixups(TCA start, TCA end, CGMeta& fixups);
-size_t relocate(RelocationInfo& rel,
-                CodeBlock& destBlock,
-                TCA start, TCA end,
-                CGMeta& fixups,
-                TCA* exitAddr);
-
-}}}
+size_t relocate(RelocationInfo&, CodeBlock&, TCA, TCA, CGMeta&, TCA*);
```

Comment:
Keep the variable names for these prototypes, since this is the API.  (I wish it were documented, but you don't need to worry about that.)